assets/base.css

```diff
@@ -2197,14 +2205,9 @@ product-info .loading-overlay:not(.hidden) ~ *,
   box-shadow: 0 0 0 -0.2rem rgb(var(--color-foreground));
 }
 
-@media screen and (min-width: 750px) {
-  .announcement-bar .slider-button--next {
-    margin-right: -3.2rem;
-  }
-
-  .announcement-bar .slider-button--prev {
-    margin-left: -3.2rem;
-  }
+.localization-wrapper {
+  grid-area: language-currency;
+  align-self: center;
```

Comment:
This shouldn't be `center` I believe. It should have: 
```css 
grid-area: language-currency;
align-self: center;
display: flex;
justify-content: flex-end;
```

Reply:
Otherwise it's aligned as we would expect/want: 
![](https://screenshot.click/06-53-nsysh-43g25.png)